Dockerfile

```diff
@@ -6,20 +6,22 @@ ARG BASE
 
 FROM ${BASE}
 
+LABEL org.opencontainers.image.authors="Torin Sandall <torinsandall@gmail.com>"
```

Comment:
Does this replace the `MAINTAINER` command completely...? 🤔

Reply:
It does. [Docker says the `MAINTAINER` instruction is deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).